duckduckgo/macos-browser
# Show privacy dashboard for duckduckgo serp

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204315037958579/f

**Description**: At the moment we don’t show the privacy dashboard in duckduckgo searches. This change will show the dashboard.

**Steps to test this PR**:
1. Perform a search and check the shield icon is present and the privacy dashboard can be accessed



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
